lib/validators/liveValidator.js

```diff
@@ -98,7 +98,8 @@ class LiveValidator {
       swaggerPaths = self.options.swaggerPaths;
       log.debug(`Using user provided swagger paths. Total paths: ${swaggerPaths.length}`);
     } else {
-      swaggerPaths = glob.sync(path.join(self.options.directory, '/**/swagger/*.json'));
+      let allJsons = path.join(self.options.directory, '/specification/**/*.json');
```

Comment:
if this works -> `'/specification/**/swagger/**/*.json'` then you don't need to do filtering twice